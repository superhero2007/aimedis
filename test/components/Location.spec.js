import React from 'react';
import { mount } from 'enzyme';

import Location from 'components/Location';

function setup() {
  return mount(<Location />);
}

describe('Location', () => {
  const wrapper = setup();

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance()).toBeNull();
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
