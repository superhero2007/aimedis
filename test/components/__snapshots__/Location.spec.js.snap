// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location should render properly 1`] = `"<div style=\\"height: 100%;\\"></div>"`;
